pkg/ports/ports.go

```diff
@@ -40,6 +41,21 @@ func IsPortAvailable(port uint16) bool {
 	return false
 }
 
+// This function checks if a service is ready on the specified port
+
+func CheckPortReady(port uint16) bool {
```

Comment:
Please rename the command to `IsPortReady` to be in line with our code style and the semantics of the function itself.

Reply:
done :+1: